README.md

```diff
@@ -86,6 +86,14 @@ cd git-machete
 sudo make install
 ```
 
+### Using Snap (Ubuntu)
```

Comment:
Pls move up, right before the PPA

Reply:
Also, snap isn't just for Ubuntu, right? At least they claim to be working on all major Linux distros